x/capability/keeper/keeper_test.go

```diff
@@ -114,16 +114,6 @@ func (suite *KeeperTestSuite) TestNewCapability() {
 	suite.Require().Nil(cap)
 }
 
-func (suite *KeeperTestSuite) TestOriginalCapabilityKeeper() {
```

Comment:
@AdityaSripal should I replace this test with something? If so, how?

Reply:
Ahh this is testing the original capability keeper in simapp since that was holding a capability for transfer. Now that simapp no longer does that, we don't need this test.

I would remove it.